requirements_lint.txt

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
just asking.
Why do we need this ?

Why not define this dependency as an extra for `dev` via `[options.extras_require]`?

`pip install -e .[dev]`

I expect that anyone that will do dev work on this repo, will need to install `ruff`.

Reply:
The pinned `.txt` is what Dependabot updates.

I'm generally not a fan of defining a `dev` extra because I don't think that this sort of tooling should be part of the public contract that `pyproject.toml` expresses.

Reply:
ok. No problem.

I was expencting dependbot to handle pyproject.toml https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#pip-and-pip-compile


> pip and pip-compile
> 
> In addition to supporting updates to requirements.txt files, Dependabot supports updates to pyproject.toml files if they follow the PEP 621 standard.